crates/core/tedge_api/src/health.rs

```diff
@@ -27,3 +27,16 @@ pub async fn send_health_status(responses: &mut impl PubChannel, daemon_name: &s
     let health_message = Message::new(&response_topic_health, health_status);
     let _ = responses.send(health_message).await;
 }
+
+pub fn get_last_will_message(daemon_name: String) -> Message {
```

Comment:
```suggestion
pub fn health_status_down_message(daemon_name: String) -> Message {
```
That's what this message is. We're just using it as a last will message.

Reply:
updated

Reply:
Minor: I prefer `health_status_down_message` as proposed by @albinsuresh rather than `get_health_status_down_message` as currently proposed.